Multisig/UI/ClaimToken/Views/TweetBox.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  TweetBox.swift
+//  Multisig
+//
+//  Created by Vitaly on 02.08.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+
+import UIKit
+
+class TweetBox: UINibView {
+    
+    @IBOutlet weak var tweetLabel: UILabel!
+    @IBOutlet weak var tweetButton: UIButton!
+
+    override func commonInit() {
```

Comment:
Is this method needed ? same code in awakeFromNib

Reply:
no, removed